### ISSUE TITLE
[APP/COM_PUBLICATIONS] A newer version of a forked publication removes the forking attribution

### ISSUE THREAD

**Issue by mdlama** [qubeshub/hubzero-cms#21]
Is this what we want?

**Comment by hayleyorndorf** [comment]
This is sticky, but I think "no." Despite the act of versioning, the user's foundation is still forked material. If there are enough changes or revisions (after some arbitrary line) the user should start a new publication, right?

If we have to decide when to drop the attribution for a version of a forked material, we have to decide if it crossed the "arbitrary line" each time this happens. I see headaches. I'd rather put it on the user to decide to start a new publication if they feel the need. 

Those are my first thoughts but I haven't thought through many use cases with it either. Will ponder.

**Comment by mdlama** [comment]
I agree with your thoughts, Hayley.  What that would mean is we would need to make it as clear as possible to the user that they need to make a new publication to drop the forked attribution.  This is something that I battled with with Serenity, actually.  It would have helped me to have read an article about this issue, so I think we ought to create one (maybe a knowledge base article?)  Let's talk more about it during the publication meeting today.
